public/app/features/dashboard/dashgrid/PanelStateWrapper.tsx

```diff
@@ -717,6 +719,7 @@ export class PanelStateWrapper extends PureComponent<Props, State> {
             isViewing={isViewing}
             alertState={alertState}
             data={data}
+            multipleErrors={this.state.multipleErrors}
```

Comment:
`multipleErrors`  it is being implemented only in the old panel header, are there plans to make the same adjustments in the new PanelHeader?  

ps, The new panel header is being developed under `newPanelChromeUI` :)

Reply:
good shout :+1: I missed that